tests/mgbench/runners.py

```diff
@@ -521,3 +541,193 @@ def get_memory_usage(self, workload):
 
     def fetch_client(self) -> BoltClient:
         return BoltClient(benchmark_context=self.benchmark_context)
+
+
+class MemgraphDocker(BaseRunner):
+    def __init__(self, benchmark_context: BenchmarkContext):
+        super().__init__(benchmark_context=benchmark_context)
+        self._directory = tempfile.TemporaryDirectory(dir=benchmark_context.temporary_directory)
+        self._vendor_args = benchmark_context.vendor_args
+        self._bolt_port = self._vendor_args["bolt-port"] if "bolt-port" in self._vendor_args.keys() else "7687"
+        self._container_name = "memgraph_benchmark"
+        self._container_ip = None
+        self._config_file = None
+
+    def _set_args(self, **kwargs):
+        return _convert_args_to_flags(**kwargs)
+
+    def start_db_init(self, message):
+        command = [
+            "docker",
+            "run",
+            "--detach",
+            "--name",
+            self._container_name,
+            "-it",
+            "-p",
+            self._bolt_port + ":" + self._bolt_port,
+            "memgraph/memgraph",
+            "--telemetry_enabled=false",
+            "--storage_wal_enabled=false",
+            "--storage_recover_on_startup=true",
+            "--storage_snapshot_interval_sec",
+            "0",
+        ]
+        command.extend(self._set_args(**self._vendor_args))
+        ret = self._run_command(command)
+
+        command = [
+            "docker",
+            "cp",
+            self._container_name + ":etc/memgraph/memgraph.conf",
+            self._directory.name + "/memgraph.conf",
+        ]
+        self._run_command(command)
+        self._config_file = Path(self._directory.name + "/memgraph.conf")
+
+        command = ["docker", "inspect", "--format", "{{ .NetworkSettings.IPAddress }}", self._container_name]
+        ret = subprocess.run(command, check=True, capture_output=True, text=True)
+        ip_address = ret.stdout.strip("\n")
+        _wait_for_server(self._bolt_port, ip=ip_address)
+
+    def stop_db_init(self, message):
+        # Stop to save the snapshot
+        command = ["docker", "stop", self._container_name]
+        self._run_command(command)
+
+        # Change config back to default
+        argument = "--storage-snapshot-on-exit=false"
+        self._replace_config_args(argument)
+        command = [
+            "docker",
+            "cp",
+            self._config_file.resolve(),
+            self._container_name + ":etc/memgraph/memgraph.conf",
```

Comment:
`/` is missing

Reply:
In the docker container using / or not using it is the same. The / is assumed. I have added it for readability.